app/views/publish/users/new.html.erb

```diff
@@ -0,0 +1,30 @@
+<% page_title = t("page_titles.users.new") %>
+<%= content_for :page_title, title_with_error_prefix(page_title, @user_form.errors.any?) %>
+
+<% content_for :before_content do %>
+  <%= govuk_back_link_to(publish_provider_users_path(params[:provider_code])) %>
+<% end %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds-from-desktop">
+    <%= form_with(model: @user_form, url: publish_provider_users_path(params[:provider_code]), local: true) do |f| %>
+      <%= f.govuk_error_summary %>
+      <span class="govuk-caption-l">Add user</span>
+      <fieldset class="govuk-fieldset">
+        <legend class="govuk-fieldset__legend govuk-fieldset__legend--l">
+          <h1 class="govuk-fieldset__heading"><%= t("page_titles.users.new") %></h1>
+        </legend>
+
+        <%= f.govuk_text_field :first_name, label: { text: "First name", size: "s" }, width: "two-thirds" %>
+        <%= f.govuk_text_field :last_name, label: { text: "Last name", size: "s" }, width: "two-thirds" %>
+        <%= f.govuk_text_field :email, label: { text: "Email address", size: "s" }, width: "full" %>
+
+        <%= f.govuk_submit %>
+      </fieldset>
+    <% end %>
+
+    <p class="govuk-body">
+      <%= govuk_link_to(t("cancel"), publish_provider_users_path(params[:provider_code])) %>
+    </p>
+  </div>
+</div>
```

Comment:
```suggestion
<div class="govuk-grid-row">
  <div class="govuk-grid-column-full">
    <%= f.govuk_error_summary %>
  </div>
</div>

<div class="govuk-grid-row">
  <div class="govuk-grid-column-two-thirds">
    <%= form_with(model: @user_form, url: publish_provider_users_path(params[:provider_code]), local: true) do |f| %>
      <fieldset class="govuk-fieldset">
        <legend class="govuk-fieldset__legend govuk-fieldset__legend--l">
          <h1 class="govuk-fieldset__heading">
            <span class="govuk-caption-l">Add user</span>
            <%= t("page_titles.users.new") %>
          </h1>
        </legend>
        <%= f.govuk_text_field :first_name, label: { text: "First name", size: "s" }, width: "two-thirds" %>
        <%= f.govuk_text_field :last_name, label: { text: "Last name", size: "s" }, width: "two-thirds" %>
        <%= f.govuk_text_field :email, label: { text: "Email address", size: "s" }, width: "full" %>
        <%= f.govuk_submit %>
      </fieldset>
    <% end %>

    <p class="govuk-body">
      <%= govuk_link_to(t("cancel"), publish_provider_users_path(params[:provider_code])) %>
    </p>
  </div>
</div>
```

Reply:
Otherwise we get this

<img width="886" alt="image" src="https://user-images.githubusercontent.com/50492247/206258071-45edd411-9d3c-4a6e-afda-33de206c3434.png">

Reply:
I've re-arranged the html to get what I think you wanted! 

Here: https://github.com/DFE-Digital/publish-teacher-training/pull/3159/commits/1d46eb6935d6204d0b75ca5a1b3148c617b6b649